contracts/test/SweepERC20TokensHarness.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+import "../sweep/SweepERC20.sol";
+
+contract SweepERC20TokensHarness is SweepERC20 {
+    function _authorizeUpgrade(address newImplementation)
+        internal
+        view
+        override
+    {
+        // blank
```

Comment:
question: is this comment needed to work around that static analyzer?

Reply:
They are in fact not required. Removed.